components/generic/ContentCard.vue

```diff
@@ -18,11 +18,24 @@
         <b-card-title>
           {{ title | truncate(90, $t('formatting.ellipsis')) }}
         </b-card-title>
-        <b-card-text
-          v-for="(text, index) in texts"
-          :key="index"
+        <time
+          v-if="datetime"
+          class="font-weight-bold pb-3"
+          data-qa="date"
+          :datetime="datetime"
```

Comment:
If you merge master into your branch, you can use localised/formatted version of the date. It is used on the blog page.